t/TestUtil.pm

```diff
@@ -12,7 +13,7 @@ sub my_default_packages_file {
 	return Parse::CPAN::Packages::Fast->_default_packages_file_interactive;
     }
 
-    my $packages_file = eval { Parse::CPAN::Packages::Fast->_default_packages_file_batch };
+    my $packages_file = getcwd() . '/t/02packages.details.txt.gz';
```

Comment:
This is a large file to be adding (1.56 MB) to a dist that is currently 8 KB. Perhaps a smaller file could demonstrate the same functionality being tested.